src/main/java/org/itsallcode/openfasttrace/importer/legacytag/LegacyTagImporterConfig.java

```diff
@@ -24,24 +24,32 @@
 
 import static java.util.Collections.emptyList;
 
+import java.nio.file.Path;
 import java.util.List;
 
 public class LegacyTagImporterConfig
 {
     private final List<PathConfig> pathConfigs;
+    private final Path basePath;
 
     public LegacyTagImporterConfig()
     {
-        this(emptyList());
+        this(null, emptyList());
```

Comment:
Using "null" this way always makes my danger sense tingle. How about an "Optional".
Alternatively a extra method to see if the path is set so that you can avoid the null checks in the calling code?

Reply:
👍 Using Optional.